sync/class.jetpack-sync-settings.php

```diff
@@ -106,6 +131,10 @@ static function update_settings( $new_settings ) {
 		}
 	}
 
+	static function is_network_setting( $setting ) {
+		return substr( $setting, 0, strlen('network_') ) === 'network_';
```

Comment:
We could simplify this:

```suggestion
		return strpos( $setting, 'network_' ) === 0;
```

Reply:
❤️ it. great suggesion